oem.php

```diff
@@ -62,25 +62,25 @@
         <h4 id="plugs">Plugs</h4>
         <p>System Settings (codenamed <a href="https://github.com/elementary/switchboard" target="_blank" rel="noopener">Switchboard</a>) uses a concept of “Plugs” to provide pluggable settings for various hardware and software concerns. For example, <a href="https://github.com/elementary/switchboard-plug-mouse-touchpad/" target="_blank" rel="noopener">Mouse & Touchpad settings</a> are provided by a plug.</p>
         <img src="images/oem/switchboard.png" alt="Switchboard" />
-        <p>OEMs are able to develop and ship custom plugs if there is special or unique hardware in the machine that would require configuration outside of what is available by default in elementary OS; for example, special sensors or input methods that are not present in most hardware and therefor not implemented by default. Of course if this configuration could be relevant to a broader set of users, contributing upstream to the <a href="https://github.com/elementary?q=switchboard-plug" target="_blank" rel="noopener">existing plugs</a> is heavily encouraged.</p>
+        <p>OEMs can develop and ship custom plugs if special or unique hardware in the machine requires configuration in addition to what is available by default in elementary OS; for example, special sensors or input methods not present in most hardware. If relevant to a broader set of users, contributing it upstream to the <a href="https://github.com/elementary?q=switchboard-plug" target="_blank" rel="noopener">existing plugs</a> is highly encouraged.</p>
         <a href="https://valadoc.org/switchboard-2.0/Switchboard.Plug.html" target="_blank" rel="noopener" class="read-more">Documentation</a>
 
         <h4 id="oem-info">OEM Info in About</h4>
-        <p>The <a href="https://github.com/elementary/switchboard-plug-about/" target="_blank" rel="noopener">About</a> plug displays system information to the user and provides several system-wide actions, such as restoring settings, reporting issues, and getting help. In addition to software information (like the OS version), it also provides a space for hardware information. By default, this is filled in with a generic image and the system’s hostname. However, OEMs can provide custom branded data for this section.</p>
+        <p>The <a href="https://github.com/elementary/switchboard-plug-about/" target="_blank" rel="noopener">About</a> plug displays system info to the user and provides several system-wide actions, such as restoring settings, reporting issues, and getting help. In addition to software info (like the OS version), it also provides a space for hardware info. By default, this is filled in with a generic image and the system’s hostname. However, OEMs can provide custom branded data for this section.</p>
```

Comment:
Again, not a fan of shorthand here

```suggestion
        <p>The <a href="https://github.com/elementary/switchboard-plug-about/" target="_blank" rel="noopener">About</a> plug displays system information to the user and provides several system-wide actions, such as restoring settings, reporting issues, and getting help. In addition to software information (like the OS version), it also provides a space for hardware information. By default, this is filled in with a generic image and the system’s hostname. However, OEMs can provide custom branded data for this section.</p>
```